src/System.Linq/src/System.Linq.csproj

```diff
@@ -4,12 +4,12 @@
   <PropertyGroup>
     <ProjectGuid>{CA488507-3B6E-4494-B7BE-7B4EEEB2C4D1}</ProjectGuid>
     <AssemblyName>System.Linq</AssemblyName>
-    <AssemblyVersion>4.2.0.0</AssemblyVersion>
+    <AssemblyVersion>4.3.0.0</AssemblyVersion>
```

Comment:
The version number should come from the dir.props please remove that property here.